src/components/Main/Main.html

```diff
@@ -20,7 +20,7 @@ <h3 class="mdc-drawer__title main__user">
           <i class="material-icons mdc-list-item__graphic" aria-hidden="true">person_outline</i>
           <span class="mdc-list-item__text">Artists</span>
         </a>
-        <hr class="mdc-list-divider">
+        <hr class="mdc-list-divider" />
```

Comment:
there is no self-closing tags in html spec - it is from xml/xhtml. In html there are just tags that do not have closing tag - like input, hr, img -- so remove that /